### PR TITLE
docs(ru): ru translation update

### PR DESCRIPTION
**Pull request by Yevhenii-Bakhmat**
Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

#### Updates of
* docker.md
* vercel.md
* faq.md
* folder-structure.md
* t3-collection.md
* next-auth.md
* prisma.md
* trpc.md
* config.ts (there were missing netlify config)
---

## Screenshots

💯

### CODE REVIEW

**Comment by Yevhenii-Bakhmat** [comment]
@AmadeusTwi @ronanru

**Comment by AmadeusTwi** [comment]
Going to review in a hour or so.

File: www/src/pages/ru/folder-structure.mdx [hunk 1]
```diff
@@ -72,72 +72,57 @@ import Diagram from "../../components/docs/folderStructureDiagram.astro";
...
+Основываясь на выбранных вами пакетах, этот маршрутизатор содержит больше или меньше маршрутов для лучшего демонстрирования использования ваших потребностей.
 
 </div>
 <div data-components="trpc">
 
-#### `src/server/trpc/router/_app.ts`
+#### `src/server/api/trpc.ts`
 
-Файл `_app.ts` используется для объединения маршрутизаторов tRPC и экспорта их в виде единого маршрутизатора, а также определений типов. Смотрите [Использование tRPC](/ru/usage/trpc#-servertrpcrouterts) для получения дополнительной информации.
+Файл `trpc.ts` - это основной файл конфигурации для вашего tRPC-бэк-энда. В нем мы:
```
Comment (ronanru) [inline www/src/pages/ru/folder-structure.mdx:114]:
```suggestion
Файл `trpc.ts` - это основной файл конфигурации для вашего tRPC-бэкэнда. В нем мы:
```

File: www/src/pages/ru/usage/trpc.md [hunk 2]
```diff
@@ -43,22 +43,27 @@ tRPC требует много шаблонного кода, который `cr
 
 ### 📄 `server/trpc/context.ts`
 
-Этот файл - это то место, где вы определяете контекст, который передается вашим tRPC процедурам. Контекст - это данные, к которым у вас есть доступ во всех ваших tRPC процедурах, и это отличное место, чтобы поместить вещи, такие как соединения с базой данных, информация об аутентификации и т.д. В create-t3-app мы используем две функции, чтобы включить использование подмножества контекста, когда у нас нет доступа к объекту запроса.
+Этот файл разделен на две части, создание контекста и инициализация tRPC:
```
Comment (ronanru) [inline www/src/pages/ru/usage/trpc.md:46]:
```suggestion
Этот файл разделен на две части: создание контекста и инициализация tRPC.
```

File: www/src/pages/ru/usage/trpc.md [hunk 3]
```diff
@@ -43,22 +43,27 @@ tRPC требует много шаблонного кода, который `cr
...
 
-### 📄 `server/trpc/router/*.ts`
+### 📄 `server/api/routers/*.ts`
+
+Это то место, где вы определяете маршруты и процедуры вашего API. Вы [создаете отдельные маршрутизаторы](https://trpc.io/docs/v10/router) для связанных процедур.
+
+### 📄 `server/api/root.ts`
 
-Это то место, где вы определяете маршруты и процедуры вашего API. Вы [создаете отдельные маршрутизаторы](https://trpc.io/docs/v10/router) для связанных процедур, а затем [сливаете](https://trpc.io/docs/v10/merging-routers) все их в единый маршрутизатор приложения в `server/trpc/router/_app.ts`.
+Здесь мы [сливаем](https://trpc.io/docs/v10/merging-routers) все под-маршрутизаторы, определенные в `routers/**` в единый маршрутизатор приложения.
```
Comment (ronanru) [inline www/src/pages/ru/usage/trpc.md:64]:
```suggestion
Здесь мы [объединяем](https://trpc.io/docs/v10/merging-routers) все под-маршрутизаторы, определенные в `routers/**` в единый маршрутизатор приложения.
```

File: www/src/pages/ru/deployment/docker.md [hunk 4]
```diff
@@ -129,7 +129,7 @@ CMD ["node", "server.js"]
 > **_Заметки_**
 >
 > - _Эмуляция `--platform=linux/amd64` может не быть необходимой после перехода на Node 18._
-> - _Посмотрите [`node:alpine`](https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine) чтобы понять, почему `libc6-compat` может быть необходим._
+> - _Посмотрите [`node:alpine`](https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine) чтобы понять, почему `libc6-compat` может быть необходим._ -_Использование пакетов базирующихся на Alpine 3.17 [может привести к ошибке](https://github.com/t3-oss/create-t3-app/issues/975). Установка `engineType = "binary"` решает проблему с Alpine 3.17, [но имеет связанные с этим затраты производительности](https://www.prisma.io/docs/concepts/components/prisma-engines/query-engine#the-query-engine-at-runtime)._
```
Comment (AmadeusTwi) [inline www/src/pages/ru/deployment/docker.md:132]:
```suggestion
> - _Посмотрите [`node:alpine`](https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine) чтобы понять, почему `libc6-compat` может быть необходим._ 
> -_Использование образов, основанных на Alpine 3.17 [может привести к проблемам с Prisma](https://github.com/t3-oss/create-t3-app/issues/975). Установка `engineType = "binary"` решает проблему с Alpine 3.17, [но имеет связанные с этим затраты производительности](https://www.prisma.io/docs/concepts/components/prisma-engines/query-engine#the-query-engine-at-runtime)._
```

File: www/src/pages/ru/deployment/vercel.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ lang: ru
 
 ## Конфигурация проекта
 
-Скорее всего Vercel настроит вашу команду сборки и каталог публикации автоматически. Однако вы также можете уточнить эту информацию вместе с другими конфигурациями, создав файл [`vercel.json`](https://vercel.com/docs/project-configuration) и включив в него следующие команды:
+Скорее всего Vercel настроит вашу команду сборки и каталог публикации автоматически. Однако вы также можете уточнить эту информацию вместе с другими конфигурациями, создав файл [`vercel.json`](https://vercel.com/docs/project-configuration) и включив в него следующие команды. **В большенстве проектов это делать не обязательно.**
```
Comment (AmadeusTwi) [inline www/src/pages/ru/deployment/vercel.md:12]:
```suggestion
Скорее всего Vercel настроит вашу команду сборки и каталог публикации автоматически. Однако вы также можете уточнить эту информацию вместе с другими конфигурациями, создав файл [`vercel.json`](https://vercel.com/docs/project-configuration) и включив в него следующие команды. **В большинстве проектов это делать необязательно.**
```
Typo.

File: www/src/pages/ru/usage/trpc.md [hunk 6]
```diff
@@ -43,22 +43,27 @@ tRPC требует много шаблонного кода, который `cr
 
 ### 📄 `server/trpc/context.ts`
```
Comment (AmadeusTwi) [inline www/src/pages/ru/usage/trpc.md:44]: Check the English docs. They might already have changed the file name due to change in file structure!

```suggestion
### 📄 `server/api/trpc.ts`
```

File: www/src/pages/ru/usage/trpc.md [hunk 7]
```diff
@@ -43,22 +43,27 @@ tRPC требует много шаблонного кода, который `cr
...
 
-### 📄 `server/trpc/trpc.ts`
+- `createInnerTRPCContext`: Это то место, где вы определяете контекст, который не зависит от запроса, например, ваше соединение с базой данных. Вы можете использовать эту функцию для [интеграционного тестирования](#sample-integration-test) или [ssg-помощников](https://trpc.io/docs/v10/ssg-helpers), где у вас нет объекта запроса.
 
-Это то место где вы инициализируете tRPC и определяете повторно используемые [процедуры](https://trpc.io/docs/v10/procedures) и [посредники](https://trpc.io/docs/v10/middlewares). По соглашению, вы не должны экспортировать весь объект `t`, а вместо этого создавать повторно используемые процедуры и посредники и экспортировать их.
+- `createTRPCContext`: Это то место, где вы определяете контекст, который зависит от запроса, например, сессия пользователя. Вы запрашиваете сессию с помощью объекта `opts.req`, а затем передаете сессию в функцию `createInnerTRPCContext` для создания окончательного контекста.
```
Comment (AmadeusTwi) [inline www/src/pages/ru/usage/trpc.md:52]:
```suggestion
- `createTRPCContext`: Здесь вы определяете контекст, который зависит от запроса: например, сессия пользователя. Вы запрашиваете сессию с помощью объекта `opts.req`, а затем передаете сессию в функцию `createInnerTRPCContext` для создания окончательного контекста.
```

**Review by AmadeusTwi — CHANGES_REQUESTED** [review]
Overall translation is good to go, but the error in the Folder Structure file name must be fixed and double-checked against similar cases.
Create-t3-app recently had bunch of changes in it's file structure, thus git-merging English docs could've lead to unexpected results like headers mismatch.

File: www/src/pages/ru/deployment/docker.md (hunk 4 above)
Comment (AmadeusTwi) [reply www/src/pages/ru/deployment/docker.md:132]: image
I think this result was not expected. It should have 4 bullet points.

File: www/src/pages/ru/usage/trpc.md (hunk 6 above)
Comment (AmadeusTwi) [reply www/src/pages/ru/usage/trpc.md:44]: image

**Comment by AmadeusTwi** [comment]
Ready to merge :)

**Review by AmadeusTwi — APPROVED** [review]

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**